### PR TITLE
6112 create home ui

### PR DESCRIPTION
**Pull request by TungPhanThanh**
Create UI tab Home.
Create fragment Home.
Create item movie layout.
https://edu-redmine.framgia.vn/issues/6112
image

### CODE REVIEW

**Comment by TungPhanThanh** [comment]
ready

**Comment by TungPhanThanh** [comment]
ready

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 <resources>
     <string name="app_name">MovieDB_35</string>
     <string name="get_started">Get Started</string>
+    <string name="title_movie">Movie</string>
+    <string name="title_category">Categories</string>
+    <string name="title_settings">Settings</string>
+    <string name="popular">Popular</string>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/values/strings.xml]: 7->10 convention 3.2.2

**Comment by TungPhanThanh** [comment]
ready

**Pull request merged**